table/tables/tables.go

```diff
@@ -427,6 +427,8 @@ func (t *TableCommon) UpdateRecord(ctx context.Context, sctx sessionctx.Context,
 		}
 	}
 
+	colIDs, row = appendMetaCol(sctx, colIDs, row)
```

Comment:
I see `appendMetaCol` is added to `UpdateRecord()` and `AddRecord()`, how about `RemoveRecord`?